File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer,Integer> freq = new HashMap<>();
        for(int num:nums){
            freq.put(num, freq.getOrDefault(num, 0)+1);
        }
        List<Map.Entry<Integer, Integer>> entries = new ArrayList<>(freq.entrySet());
        entries.sort((e1, e2)->e2.getValue().compareTo(e1.getValue()));
        int[] res = new int[k];
        System.out.print(entries);
        for(int i=0;i<k;i++){
            System.out.print(entries.get(i));
            res[i] = entries.get(i).getKey();
        }
        return res;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        hs = set(nums)
        mx = 0
        for num in hs:
            if num-1 not in hs:
                cur = num 
                cnt = 1
                while cur+1 in hs:
                    cur+=1
                    cnt+=1
                mx = max(mx, cnt)
        return mx
        